Libs/MRML/Widgets/qMRMLColorModel.cxx

```diff
@@ -325,6 +342,49 @@ void qMRMLColorModel::updateItemFromColor(QStandardItem* item, int color, int co
       }
     case qMRMLColorModel::LabelColumn:
       item->setText(colorName);
+      // check for terminology
+      if (this->mrmlColorLogic() &&
+          this->mrmlColorLogic()->TerminologyExists(d->MRMLColorNode->GetName()))
+        {
+        const char *lutName = d->MRMLColorNode->GetName();
+        std::string category = this->mrmlColorLogic()->GetSegmentedPropertyCategoryCodeMeaning(color, lutName);
+        std::string type = this->mrmlColorLogic()->GetSegmentedPropertyTypeCodeMeaning(color, lutName);
+        std::string typeMod = this->mrmlColorLogic()->GetSegmentedPropertyTypeModifierCodeMeaning(color, lutName);
+        std::string region = this->mrmlColorLogic()->GetAnatomicRegionCodeMeaning(color, lutName);
+        std::string regionMod = this->mrmlColorLogic()->GetAnatomicRegionModifierCodeMeaning(color, lutName);
+        QString terminology = QString("Terminology:");
+        // only show the not empty terminology terms
+        if (!category.empty())
+          {
+          terminology = terminology + QString("\nCategory: %1").arg(category.c_str());
```

Comment:
I would use here full attribute names exactly as they are defined in DICOM: SegmentedPropertyCategory, SegmentedPropertyType + Modifier, AnatomicRegion + Modifier